### PR TITLE
feat(header): implement new design as per issue #888

### PR DESCRIPTION
**Pull request by ankitmukhia**
### PR Fixes:
- Redesigned everything that was mentioned
- Implemented the necessary changes

Resolves #[#888] 

### Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] I assure there is no similar/duplicate pull request regarding same issue

### CODE REVIEW

**Comment by siinghd** [comment]
@ankitmukhia fix pnpm lock

File: src/app/(main)/(pages)/watch-history/page.tsx [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+import { Fragment } from 'react';
+
+export type TWatchHistory = VideoProgress & {
+  content: Content & {
+    parent: { id: number; courses: CourseContent[] } | null;
+    VideoMetadata: { duration: number | null } | null;
+  };
+};
+
+const formatWatchHistoryDate = (date: Date) => {
```
Comment (siinghd) [inline src/app/(main)/(pages)/watch-history/page.tsx:15]: don't we have dayjs? try to use dayjs and avoid creating these functions

**Comment by siinghd** [comment]
@ankitmukhia fix lock and these review other than that looks good

**Comment by ankitmukhia** [comment]
> @ankitmukhia fix lock and these review other than that looks good

fixed it...

**Comment by ankitmukhia** [comment]
> @ankitmukhia fix lock and these review other than that looks good

## kindly review this at your convenience.

## 1
Screenshot from 2024-08-19 04-01-21

## 2
Screenshot from 2024-08-19 04-01-36

## 3
Screenshot from 2024-08-19 04-01-48
ets/2104701c-56c8-4f28-9546-2214655c8945)

File: src/app/(main)/(pages)/watch-history/page.tsx (hunk 1 above)
Comment (ankitmukhia) [reply src/app/(main)/(pages)/watch-history/page.tsx:15]: i'll make sure.

**Comment by hkirat** [comment]
/bounty $550
Thankyou!
This is contingent after Due delligence to make sure the code wasn't copied from another PR (Don think it was but just putting it out here)

**Comment by ankitmukhia** [comment]
> /bounty $550 Thankyou! This is contingent after Due delligence to make sure the code wasn't copied from another PR (Don think it was but just putting it out here)

thx for the offer! Just wanna say the code’s all mine, didn’t copy from any other PR

**Pull request merged**